### ISSUE TITLE
Issue with WorkManager for Local Notifications on iOS

### ISSUE THREAD

**Issue by burhanaksendir** [NedaaDevs/nedaa#51]
As-salamu alaykum brothers,

First of all, thank you for this useful project. I'm working with WorkManager for local notifications on Android, but it doesn't seem to work on iOS. Have you encountered this issue as well? I would greatly appreciate any assistance you can provide on this matter.

**Comment by Amjad50** [comment]
Walaikumassalam.

Can you share what's the issue you are facing?

**Comment by burhanaksendir** [comment]
@Amjad50  Thanks. I downloaded your project and tested it on iOS and Android devices. The workmanager you used works flawlessly on Android. However, I did not see it working on iOS. Is there any extra configuration I need to do?

**Comment by F2had** [comment]
> @Amjad50 Thanks. I downloaded your project and tested it on iOS and Android devices. The workmanager you used works flawlessly on Android. However, I did not see it working on iOS. Is there any extra configuration I need to do?

Hi, 
do you mean that worker manager is not triggering the job in the background?

**Comment by burhanaksendir** [comment]
@F2had 
 Yes, I'm referring to implementing a periodic task in the background on the Android side to update local notifications every 2 days. Thank you for your response. I managed to get it working as desired in the end. I've submitted a PR

**Comment by F2had** [comment]
> @F2had 
> 
>  Yes, I'm referring to implementing a periodic task in the background on the iOS side to update local notifications every 2 days. Thank you for your response. I managed to get it working as desired in the end. I've submitted a PR

@burhanaksendir 
Glad you managed to do it & thank you Burhan for the PR. Will check it out once I get on my laptop.

**Comment by burhanaksendir** [comment]
@F2had  Thanks. Please follow these steps:  
flutter clean
flutter pub cache,
and
flutter build ios. 

Then try testing again.
